mitosheet/mitosheet/tests/test_utils.py

```diff
@@ -595,7 +595,8 @@ def export_to_file(
                 'params': {
                     'type': type,
                     'sheet_indexes': sheet_indexes,
-                    'file_name': file_name
+                    'file_name': file_name,
+                    'export_formatting': True
```

Comment:
Let's make this a parameter to the function, and make it default to True. 

Then, can we add a test to make sure things still work (and you don't get formatting) when this is set to False!